.github/workflows/apt.yml

```diff
@@ -180,6 +201,10 @@ jobs:
       - uses: actions/download-artifact@v4
         with:
           name: packages-${{ matrix.rake-job }}
+      - uses: actions/download-artifact@v4
+        with:
+          name: v6-packages-${{ matrix.rake-job }}
+          path: v6repositories
```

Comment:
Don't we need this for `yum.yml` as well?

Reply:
Oops, it should be so.